plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergModule.java

```diff
@@ -78,5 +81,7 @@ public void configure(Binder binder)
 
         Multibinder<Procedure> procedures = newSetBinder(binder, Procedure.class);
         procedures.addBinding().toProvider(RollbackToSnapshotProcedure.class).in(Scopes.SINGLETON);
+
+        newOptionalBinder(binder, ConnectorAccessControl.class).setDefault().to(AllowAllAccessControl.class).in(Scopes.SINGLETON);
```

Comment:
are there further changes planned here? 

If not, would it be better to introduce config-based binding at this point? Otherwise, in future, if we introduce hive-like config-based bindings here, custom implementations will still keep trying to bind `ConnectorAccessControl` themselves without looking at the config, and we end up with different mechanisms for binding (for custom implementations). It doesn't sound THAT bad, but curious to know your thoughts.